sharepoint/sharepoint-ps/sharepoint-online/Get-SPORestrictedAccessForSitesInsights.md

```diff
@@ -0,0 +1,244 @@
+---
+external help file: sharepointonline.xml
+Module Name: Microsoft.Online.SharePoint.PowerShell
+online version: https://learn.microsoft.com/powershell/module/sharepoint-online/get-sporestrictedaccessforsitesinsights
+applicable: SharePoint Online
+title: Get-SPORestrictedAccessForSitesInsights
+schema: 2.0.0
+author: itambakhe
+ms.author: itambakhe
+ms.reviewer:
+manager:
+---
+
+# Get-SPORestrictedAccessForSitesInsights
+
+## SYNOPSIS
+
+This cmdlet enables the administrator to check status of all active and available reports about insights on sites protected and access denials by Restricted access control policy when no report ID is present and to view or download the report if report ID is present.
+
+## SYNTAX
+
+```powershell
+Get-SPORestrictedAccessForSitesInsights -RACProtectedSites [-ReportId <Guid>] [-Action <ActionType>] [-InsightsSummary <Boolean>]
+```
+
+```powershell
+Get-SPORestrictedAccessForSitesInsights -ActionsBlockedByPolicy [-ReportId <Guid>] [-Content <ContentType>] [-Action <ActionType>]
```

Comment:
The cmdlet implementation could be improved to use ParameterSets for clarity. Please work with Dev to do this as an improvement as a followup change.